### PR TITLE
Kustomize em-npa

### PR DESCRIPTION
**Pull request by yogesh-hullatti**
https://tools.hmcts.net/jira/browse/EM-3330

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -127,7 +127,7 @@ k8s/**/common/dm-store/dm-store.yaml @hmcts/evidence
 k8s/namespaces/em/ @hmcts/evidence
 k8s/**/common/em/identity.yaml @hmcts/evidence
 k8s/**/common/em/em-anno.yaml @hmcts/evidence
-k8s/**/common/em/em-npa.yaml @hmcts/evidence
+k8s/namespaces/em/em-npa/em-npa.yaml @hmcts/evidence
```
Comment (adusumillipraveen) [inline CODEOWNERS:130]: You should have full control on the em directory
```suggestion
```

File: k8s/namespaces/em/em-npa/prod.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: em-npa
+spec:
+  values:
+    java:
```
Comment (adusumillipraveen) [inline k8s/namespaces/em/em-npa/prod.yaml:8]: Don't u need to set these? https://github.com/hmcts/cnp-flux-config/pull/7647/files#diff-75adbbb47bc22a3aafdb1984aef03e8f917b283a709ff161b6c36e174d92f7deL23-L24

**Review by adusumillipraveen — COMMENTED** [review]
Minor comments

**Review by adusumillipraveen — APPROVED** [review]

File: k8s/namespaces/em/em-npa/prod.yaml (hunk 2 above)
Comment (yogesh-hullatti) [reply k8s/namespaces/em/em-npa/prod.yaml:8]: We are ok with the default values at the moment.

**Review by benouaer — APPROVED** [review]

**Pull request merged**